.travis.yml

```diff
@@ -22,9 +23,6 @@ after_script:
   - 'npm install coveralls && cat coverage/lcov.info | coveralls'
 
 after_success:
-  - 'curl -Lo travis_after_all.py https://git.io/travis_after_all'
-  - python travis_after_all.py
-  - 'export $(cat .to_export_back) &> /dev/null'
```

Comment:
What is that?

Reply:
This repository uses [semantic-releast](https://github.com/semantic-release/semantic-release) and this was needed for it. With version 6, it was not necessary anymore: https://github.com/semantic-release/semantic-release/releases/tag/v6.0.0

Reply:
Thanks 👍